### ISSUE TITLE
Projects .vir file is zero filled

### ISSUE THREAD

**Issue by maffeirw** [smeighan/nutcracker#42]
When I use the any effect in projects and I generate a Vixen .vir file, it's zero filled. Below is the result I get from my Fire (Arthur Brown) Project (for EVES_NG) even though the  EVES_NG~HALLOWEENCHASETOPEAK effect generates a Vixen .vir file without problem when run on its own.

checking NC Files

phrase  HALLOWEENCHASETOPEAK    workarea/maffeirw~~214.nc   INVALID     (NCArray[1]==(numEntries_3),(==(206_3)
phrase  None            

Erasing gaps and joining effects

phrase  None    Adding 214 zeros from frame 0 to frame 213
phrase  None    Adding 3164 zeros from frame 214 to frame 3377
Writing to workarea/maffeirw~69~master.nc

**Comment by kgustafson** [comment]
This generated a zero filled effect due to an error in the size of the arrays (See the INVALID comment above).  This may be an old error that was fixed as we worked through some of the issue with single strand effects.  I will try to recreate this error on your account.  Realize that when nutcracker encounters an error (such as the blank result that it received when it tried to generate the single strand effect) it 'punts' and fills that effects with zeros.  The program does extensive error checking before joining the effects together in the master file.
